src/View/ImageFusion/UITransferROIWindow.py

```diff
@@ -0,0 +1,635 @@
+        transfer_roi_window_instance.setWindowTitle(
+            _translate("TransferRoiWindowInstance",
+                       "OnkoDICOM - Transfer Region of Interest"))
+        self.add_suffix_checkbox.setText(
+            _translate("AddSuffixCheckBox", "Add Suffix"))
+        self.patient_A_label.setText(
+            _translate("PatientAROILabel", "First Image Set ROIs"))
+        self.patient_B_label.setText(
+            _translate("PatientBROILabel", "Second Image Set ROIs"))
+        self.transfer_all_rois_to_patient_B_button.setText(
+            _translate("ROITransferToBButton", "")
+        )
+        self.transfer_all_rois_to_patient_A_button.setText(
+            _translate("ROITransferToAButton", "")
+        )
+        self.save_button.setText(
+            _translate("SaveButton", "Save")
+        )
+        self.reset_button.setText(
+            _translate("ResetButton", "Reset")
+        )
+
+    def init_layout(self):
+        """
+        Initialize the layout for the Transfer ROI Window.
+        """
+        if platform.system() == 'Darwin':
+            self.stylesheet_path = "res/stylesheet.qss"
+        else:
+            self.stylesheet_path = "res/stylesheet-win-linux.qss"
+        stylesheet = open(resource_path(self.stylesheet_path)).read()
+        window_icon = QIcon()
+        window_icon.addPixmap(QPixmap(resource_path("res/images/icon.ico")),
+                              QIcon.Normal, QIcon.Off)
+        self.transfer_roi_window_instance.setObjectName(
+            "TransferRoiWindowInstance")
+        self.transfer_roi_window_instance.setWindowIcon(window_icon)
+
+        # Creating a grid layout to hold all elements
+        self.transfer_roi_window_grid_layout = QGridLayout()
+        self.transfer_roi_window_grid_layout.setColumnStretch(0, 1)
+        self.transfer_roi_window_grid_layout.setColumnStretch(1, 1)
+        self.transfer_roi_window_grid_layout.setColumnStretch(2, 1)
+
+        self.init_patient_labels()
+        self.init_transfer_arrow_buttons()
+        self.init_patient_A_initial_roi_list()
+        self.init_patient_B_rois_to_A_layout()
+        self.init_patient_A_rois_to_B_layout()
+        self.init_patient_B_initial_roi_list()
+        self.init_add_suffix_checkbox()
+        self.init_save_and_reset_button_layout()
+
+        # Create a new central widget to hold the grid layout
+        self.transfer_roi_window_instance_central_widget = QWidget()
+        self.transfer_roi_window_instance_central_widget.setLayout(
+            self.transfer_roi_window_grid_layout)
+        self.retranslate_ui(self.transfer_roi_window_instance)
+        self.transfer_roi_window_instance.setStyleSheet(stylesheet)
+        self.transfer_roi_window_instance.setCentralWidget(
+            self.transfer_roi_window_instance_central_widget)
+        QtCore.QMetaObject.connectSlotsByName(
+            self.transfer_roi_window_instance)
+
+    def init_transfer_arrow_buttons(self):
```

Comment:
Just curious, the buttons to transfer the ROI - transfers all the ROIs. Is it possible to transfer just one ROI using the platipy library? If so, should we give the user the option just to transfer one ROI?

Reply:
You can transfer one ROI by double-clicking the ROI name that you want to transfer. The left and right buttons are just a quick way to transfer all ROIs at once haha. I will add text to the button to make it clearer i guess

Reply:
Didn't know that. Is it too much trouble to have another way to select only 1 ROI? (Just another buttoned with 1 arrow symbol instead of double)

Reply:
I think it saves more time for double click in a ROI name to transfer, rather than click on the ROI name, then click on a button to transfer it. The demo also came through with the product owner and he seemed to like the idea, I guess we will see when he actually uses it.